pkg/mailhog/client.go

```diff
@@ -0,0 +1,93 @@
+package mailhog
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+type Receipient struct {
+	Address string `json:"Address,omitempty"`
+	Name    string `json:"Name,omitempty"`
+}
+
+type Message struct {
+	ID      string       `json:"ID,omitempty"`
+	From    Receipient   `json:"From,omitempty"`
+	To      []Receipient `json:"To,omitempty"`
+	Subject string       `json:"Subject,omitempty"`
+}
+
+type Response struct {
+	Messages []Message `json:"messages,omitempty"`
+}
+
+func NewClient(host string, port int, timeout time.Duration) *Client {
+	return &Client{
+		addr: net.JoinHostPort(host, strconv.Itoa(port)),
+		cl: &http.Client{
+			Timeout: timeout,
+		},
+	}
+}
+
+type Client struct {
+	addr string
+	cl   *http.Client
```

Comment:
short abbreviations as cl are quite hard to hold in mind, especially in nested statements, like `c.cl.Do`

Reply:
@Hukyl 
see https://github.com/GenesisEducationKyiv/software-engineering-school-4-0-hrvadl/pull/3#discussion_r1646277498